packages/core/theme/src/components/tabs.ts

```diff
@@ -159,6 +160,22 @@ const tabs = tv({
         tabContent: "transition-none",
       },
     },
+    tabPosition: {
+      top: {},
+      start: {
+        tabList: "flex-col",
+        panel: "py-0 px-3",
+        wrapper: "flex",
+      },
+      end: {
+        tabList: "flex-col",
+        panel: "py-0 px-3",
+        wrapper: "flex flex-row-reverse",
+      },
+      bottom: {
+        wrapper: "flex flex-col-reverse",
+      },
+    },
```

Comment:
@winchesHe could you please check the RTL support?

Reply:
ok, I will check it

Reply:
<img width="936" alt="image" src="https://github.com/nextui-org/nextui/assets/96854855/f08a3479-0e73-4dfc-ae65-d66b473baf31">

<img width="936" alt="image" src="https://github.com/nextui-org/nextui/assets/96854855/603c9e86-efa7-4aa1-af2c-d5b793adc9df">

I checked and it supports RTL very well

Reply:
space between the text and radio maybe had some style issue in RTL